### DISCUSSION TITLE
Prevent Drag Column Sorting on SPECIFIC columns?

### DISCUSSION THREAD

**Discussion by rchawla-alphabyte** [serenity-is/Serenity#5990] [Help]
Say for example my grid has 10 columns. 

I want to ensure specific columns CANNOT be dragged and sorted within the grid.

How would I accomplish this?

**Comment by edwardch** [comment]
Haven't tried to disable column dragging, but to disable sorting you can add `[Sortable(false)]` above the field in yourRow.cs.

**Comment by rchawla-alphabyte** [comment]
Unfortunately, that does not solve my issue - any other suggestions would be appreciated!
